### PR TITLE
Update Editors

### PR DESCRIPTION
**Pull request by npm1**
This PR moves Shubhie to former editor and adds 3 new editors in alphabetical order.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/paint-timing/pull/85.html" title="Last updated on Apr 20, 2020, 1:13 PM UTC (b7a01b8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/paint-timing/85/ecf619e...b7a01b8.html" title="Last updated on Apr 20, 2020, 1:13 PM UTC (b7a01b8)">Diff</a>

### CODE REVIEW

File: painttiming.bs [hunk 1]
```diff
@@ -6,7 +6,10 @@ Level: 1
 ED: https://w3c.github.io/paint-timing/
 TR: https://www.w3.org/TR/paint-timing/
 Status: ED
-Editor: Shubhie Panicker, Google https://google.com, panicker@google.com
+Editor: Nicolás Peña Moreno, Google https://google.com, npm@chromium.org
+        Noam Rosenthal
```
Comment (npm1) [inline painttiming.bs:10]: @noamr I don't know if you want to provide a company and/or contact email here.

Comment (noamr) [reply painttiming.bs:10]: Noam Rosenthal, invited expert <noam@webkit.org>

(Is it customary to write invited expert in these? That’s my affiliation here..., leave out if it’s not the custom)

File: painttiming.bs [hunk 2]
```diff
@@ -6,7 +6,10 @@ Level: 1
 ED: https://w3c.github.io/paint-timing/
 TR: https://www.w3.org/TR/paint-timing/
 Status: ED
-Editor: Shubhie Panicker, Google https://google.com, panicker@google.com
+Editor: Nicolás Peña Moreno, Google https://google.com, npm@chromium.org
```
Comment (npm1) [inline painttiming.bs:9]: BTW IDK what's the convention for editor order so if you prefer another order let me know.

Comment (noamr) [reply painttiming.bs:9]: Whatever you chose is fine

File: painttiming.bs [hunk 3]
```diff
@@ -6,7 +6,10 @@ Level: 1
...
+        Noam Rosenthal
+        Yoav Weiss, Google https://google.com, yoavweiss@chromium.org
```
Comment (yoavweiss) [inline painttiming.bs:11]: Looked at my [list of commits](https://github.com/w3c/paint-timing/commits?author=yoavweiss), and I don't think it justifies adding me as an editor. While I did do a bunch of [PR reviews](https://github.com/w3c/paint-timing/search?q=reviewed-by%3Ayoavweiss&type=Issues), I'm happy with you two being the active editors.

File: painttiming.bs (hunk 1 above)
Comment (yoavweiss) [reply painttiming.bs:10]: I think it can help clarify your affiliation, so would be good to add it.

File: painttiming.bs (hunk 3 above)
Comment (npm1) [reply painttiming.bs]: Ok, removed you

**Pull request merged**